docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -588,23 +594,24 @@ Stomach cancer screening algorithm was derived from the 2019 guidelines from the
 | Pre-cancer            | H. pylori negative (-)     | H. pylori positive (+)    |
 | States                | from screening test        | from screening test       |        
 +-----------------------+----------------------------+---------------------------+
-|   atrophy -           | Branch 1                   |  Branch 2                 |
+|  atrophy -            | Branch 1 (A)               |  Branch 2 (B)             |
 |                       | repeat ABC every 5 years   |  endoscopy every 3 years  |
 +-----------------------+----------------------------+---------------------------+                                                   
-|   atrophy +           | Branch 4                   | Branch 3                  |
+|  atrophy +            | Branch 4 (D)               | Branch 3  (C)             |
 |                       | endoscopy every 1 year     | endoscopy every 2 years   |          
 +-----------------------+----------------------------+---------------------------+          
 
 
-H. pylori antibiody test [Chin Med J (Engl) 2018]
+H. pylori antibiody test [Chen Chin Med J (Engl) 2018]
 
   - sensitivity 91.2%
   - specificity 97.4% 
 
-Serum pepsinogen test [Chin Med J (Engl) 2018]
+Serum pepsinogen test [Miki Gastric Cancer 2006]
 
-  - because the incidence of gastric cancer is determined by true H. pylori status and not by atrophic state, we do not need to apply test accuracy for atrophy. The atrophic state identified in model 2 determines frequency of screening.   
+  - For **S** population: because the incidence of gastric cancer for the cancer-free susceptible population is determined by true H. pylori status and not by atrophic state, we do not apply test accuracy for atrophy for the S population. The age-specific prevalence of atrophic states was obtained from serum pepsinogen test [Wang Scientific reports 2005] among cancer-free population and is thus the test-positive prevalence of atrophy among our S population. The test-positive prevalence of atrophy identified in model 2 determines frequency of screening for the S population. (In reality, those who were determined to be in group B-D by ABC screening would be referred for endoscopy and their endoscopic follow-up frequency will be determined by the outcome of the endoscopy. However, we do not have the data to use the true pre-cancer states to determine screening frequency)    
 
+  - For **PC** population: All simulants initialized in the pre-clinical cancer state are assigned to be atrophy positive. Their H. pylori status is the fraction of H. pylori+ among atrophy (f_atrophy+) for that age group. When simulants attend their first ABC screening, PC cancers will be detected with 77% sensitivity. Those who are false negatives (23%) will be assigned into branch 1 (group A) or branch 2 (group B) depending on their H. pylori status for continued surveillance. (Their H. pylori status helps to determine their screening frequency in case of false negative.) 
```

Comment:
Can we use this sensitivity and specificity, but not correlate atrophy and pre-clinical cancer in the initialization?

Reply:
will there be false positives?  what value should I use for specificity?

Reply:
we are using 98% as specificity as the combined specificity after endoscopy confirmation.